modules/rest-api/pages/upload-retrieve-root-cert.adoc

```diff
@@ -52,10 +67,10 @@ curl -X GET
   -u <username>:<password>
```

Comment:
--cacert only makes sense when https is used. Please remove --cacert from the examples where http is used.
I also suggest we remove --cacert from description below.

Reply:
I may not be quite sure what to do here. So, I think my error was that 'http' rather than 'https' was specified in the curl syntax section (this conflicted with the description, which specified secure access only). So, I have changed 'http' to 'https' - does that in itself fix the problem? Or should I do more? Thanks.

Reply:
I think we have bigger problem here.
First of all, it's not clear to me what GET /pools/default has to do with "Upload and Retrieve the Root Certificate"
Also since all other examples use http it's not clear to me why this particular one suddenly uses https.
Also description of --cacert arg belongs to curl manual. It has nothing to do with couchbase.

I suggest we remove all mentions of GET /pools/default from this page. This will also remove the example in question and the description of --cacert.
If you have a reason to keep GET /pools/default, than I suggest to change https -> http (to be consistent with other examples), and remove --cacert from example and from description below.

Reply:
I have removed all references to GET /pools/default from this page.